help/de/Help_ch_Account-Actions.xml

```diff
@@ -701,90 +803,121 @@
-                      (ohne die Anführungszeichen) suchen.
-                    </para>
-                  </listitem>
-
-                  <listitem>
-                    <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
-                      <code>gnc-fq-update</code> aus. Dadurch wird eine Perl CPAN-Update-Sitzung
-                      gestartet, die ins Internet geht und das Modul
-                      <application>Finance::Quote</application> und seine Abhängigkeiten auf Ihrem
-                      System installiert. Das Programm gnc-fq-update ist interaktiv, bei den meisten
-                      Systemen sollten Sie jedoch die erste Frage mit <quote>Nein</quote>
-                      beantworten können: <quote>Sind Sie bereit für die manuelle Konfiguration?
-                      [ja]</quote> und die Aktualisierung wird automatisch fortgesetzt von diesem
-                      Punkt an.
-                    </para>
-                  </listitem>
-                </orderedlist>
-              </listitem>
-            </itemizedlist>
+                <listitem>
+                  <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
+                    allgemeinere Anweisungen probiern:
+                  </para>
 
-            <para>Nachdem die Installation abgeschlossen ist, sollten Sie das Testprogramm <code>gnc-fq-check</code>,
-              das mit &app; verteilt wird, im gleichen Verzeichnis ausführen, um zu testen, ob
-              <application>Finance::Quote</application> installiert ist und richtig funktioniert.
+                  <orderedlist spacing="compact">
+                    <listitem>
+                      <para>Schließen Sie alle &app;-Anwendungen, die Sie gerade ausführen.
+                      </para>
+                    </listitem>
+
+                    <listitem>
+                      <para>Lokalisieren Sie den Ordner, in dem &app; installiert ist, indem Sie nach
+                        <command>gnc-fq-update</command> suchen.
+                      </para>
+                    </listitem>
+
+                    <listitem>
+                      <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
+                        <command>gnc-fq-update</command> aus. Dadurch wird eine Perl
+                        CPAN-Update-Sitzung gestartet, die ins Internet geht und das Modul
+                        &app-fq; und seine Abhängigkeiten auf
+                        Ihrem System installiert. Das Programm
+                        <command>gnc-fq-update</command> ist interaktiv, bei den meisten
+                        Systemen sollten Sie jedoch die erste Frage mit <quote>Nein</quote>
+                        beantworten können: <quote>Sind Sie bereit für eine händische
```

Comment:
<computeroutput> oder <prompt>login:</prompt>

Eingepackt in <screen> kannst du damit ganze Sitzungen darstellen.